lib/Extension/LanguageServerReferenceFinder/Tests/Unit/Handler/TypeDefinitionHandlerTest.php

```diff
@@ -50,24 +46,19 @@ public function testGoesToSingleType(): void
         $this->assertInstanceOf(Location::class, $location);
         $this->assertEquals(self::EXAMPLE_URI, $location->uri);
         $this->assertEquals(2, $location->range->start->character);
+        $this->assertEquals(5, $location->range->end->character);
     }
 
     public function testGoesToMultipleTypes(): void
     {
         $locations = [
             new TypeLocation(
                 TypeFactory::class('Foobar'),
-                new PhpactorLocation(
-                    TextDocumentUri::fromString(self::EXAMPLE_URI),
-                    ByteOffset::fromInt(2)
-                )
+                PhpactorLocation::fromPathAndOffsets(self::EXAMPLE_URI, 2, 2),
             ),
             new TypeLocation(
                 TypeFactory::class('Barfoo'),
-                new PhpactorLocation(
-                    TextDocumentUri::fromString(self::EXAMPLE_URI),
-                    ByteOffset::fromInt(2)
-                )
+                PhpactorLocation::fromPathAndOffsets(self::EXAMPLE_URI, 2, 2),
```

Comment:
maybe add `fromPathAndOffset` for locations at a single offset

Reply:
Done.